packages/f-services/src/validations.js

```diff
@@ -36,8 +36,13 @@ const isValidPostcode = postcode => POSTCODE_REGEX.test(postcode);
  */
 const meetsCharacterValidationRules = value => /^[\u0060\u00C0-\u00F6\u00F8-\u017Fa-zA-Z-' ]*$/.test(value);
 
+const PHONE_REGEX = { 'en-GB': /^(\+(44))?[0-9]{10}|(\+(44))?[0-9]{11}$/ };
```

Comment:
You might find that the `|` operator here includes `^` and `$` as part of the alternate patterns. In other words, the pattern will accept anything that *starts* with 10 digits (optionally preceded by `+44`), or anything that *ends* with 11 digits (also optionally preceded by `+44`)?

I think this could be a range, as aside from the line end/line start markers I can't see anything different between the patterns otherwise?
```suggestion
const PHONE_REGEX = { 'en-GB': /^(\+(44))?[0-9]{10,11}$/ };
```

Reply:
Sounds good to me, thanks! I've updated it 🙂